admin/src/containers/Login/Login.js

```diff
@@ -21,7 +21,7 @@ export class LoginContainer extends React.Component {
         'Content-Type': 'application/json'
       }
     }).then(async res => {
-      session.set(await res.json());
+      session.set({ ...(await res.json()), username });
```

Comment:
Ideally, we'd return that data along with the token from the back-end. Otherwise, we could have the Session method accept 2 parameters: `authData` and `username` which would then store username separately. Do you want a refactor?

Reply:
I like the backend approach, let's keep this code with that on mind 👍